packages/apps/human-app/frontend/src/modules/worker/profile/components/buttons/start-kyc-btn.tsx

```diff
@@ -0,0 +1,29 @@
+import { t } from 'i18next';
+import { Button } from '@/shared/components/ui/button';
+import { useAuthenticatedUser } from '@/modules/auth/hooks/use-authenticated-user';
+import { useStartKyc } from '../../hooks';
+
+export function StartKycBtn() {
+  const { user } = useAuthenticatedUser();
+  const { isKYCInProgress, kycStartIsPending, startKYC } = useStartKyc();
+
+  if (isKYCInProgress) {
+    return (
+      <Button disabled fullWidth variant="contained">
+        {t('worker.profile.KYCInProgress')}
+      </Button>
+    );
+  }
+
+  return (
+    <Button
+      disabled={user.status !== 'active'}
```

Comment:
We probably don't need this status check because it's about email verification and it's done in a different place